components/raftstore/src/store/fsm/peer.rs

```diff
@@ -717,11 +720,13 @@ where
                 }
             }
             CasualMessage::SnapshotGenerated => {
+                // [PerformanceCriticalPath]?? should it be done in background thread?
```

Comment:
It doesn't block.

Reply:
You're right. ping is not a blocking API. Thanks.